File: src/00/sigma_core/src/ontology_gen/01/in_grid.rs
// SSoT: file:///Users/s0fractal/OMEGA/src/ontology/spatial/in_grid.md
#![allow(unused_imports)]
use super::super::L00::*;

pub fn in_grid(x: i32, y: i32) -> bool {
    x >= 0 && x < GRID_W && y >= 0 && y < GRID_H
}

File: src/_as/01/in_grid.ts
// SSoT: file:///Users/s0fractal/OMEGA/src/ontology/spatial/in_grid.md
import { GRID_W, GRID_H } from "../00/mod";

@inline
export function in_grid(x: i32, y: i32): bool {
return x >= 0 && x < GRID_W && y >= 0 && y < GRID_H;
}
